go/src/koding/kites/config/konfig.go

```diff
@@ -55,6 +55,17 @@ type Konfig struct {
 	Debug bool `json:"debug,omitempty"`
 }
 
+// KodingBaseURL specifies the base url for koding
+func (k *Konfig) KodingBaseURL() string {
+	u, err := url.Parse(k.KloudURL)
+	if err != nil {
+		return ""
+	}
+
+	u.Path = ""
+	return u.String()
```

Comment:
could we `return u.Scheme + "://" + u.Host` 

if `k.KloudURL` had query/user/fragment parts, this wouldn't work.